pkg/ddevapp/config.go

```diff
@@ -973,8 +973,11 @@ func (app *DdevApp) RenderComposeYAML() (string, error) {
 	extraWebContent := "\nRUN mkdir -p /home/$username && chown $username /home/$username && chmod 600 /home/$username/.pgpass"
 	extraWebContent = extraWebContent + "\nENV NVM_DIR=/home/$username/.nvm"
 	if app.NodeJSVersion != nodeps.NodeJSDefault {
-		extraWebContent = extraWebContent + "\nRUN npm install -g n"
-		extraWebContent = extraWebContent + fmt.Sprintf("\nRUN n install %s && ln -sf /usr/local/bin/node /usr/local/bin/nodejs", app.NodeJSVersion)
+		extraWebContent = extraWebContent + fmt.Sprintf(`
+RUN npm install -g n
```

Comment:
It's a little confusing that we try to do this at build time and then later during start.sh. But it seems reasonable.

```suggestion
RUN npm install -g n || true
```

Reply:
This is a different thing, we run only `n install` during `start.sh`.

Maybe we should install it on `ddev-webserver` so it's available right away?

Reply:
I agree, we should have `n` already installed in ddev-webserver.